### PR TITLE
feat: oss banner, formatting, clearer owlbert pics

### PR DESCRIPTION
**Pull request by kanadgupta**
## 🧰 Changes

- [x] Added our OSS banner and surfaced it in a few places[^1]
- [x] Added `prettier` and `@readme/eslint-config` as deps and added a formatting check
- [x] Used clearer versions of our Owlbert images and ensured `https` across the board

## 🧬 QA & Testing

Do tests pass and do the changes look good?

[^1]: The relative reference might create issues for [this page](https://github.com/readmeio?view_as=public), but we'll find out.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 | 🚥 Fix RM-XXX |
```
Comment (erunion) [inline .github/PULL_REQUEST_TEMPLATE.md]: Thoughts on renaming this? I've been thinking about changing this for a while now

```suggestion
| 🚥 Fixes RM-XXX |
```

Comment (kanadgupta) [reply .github/PULL_REQUEST_TEMPLATE.md]: let's do it

Comment (kanadgupta) [reply .github/PULL_REQUEST_TEMPLATE.md]: since this is applying on all OSS repos, thoughts on removing the `RM-` prefix?

Comment (erunion) [reply .github/PULL_REQUEST_TEMPLATE.md]: 100%, having that in those repos has always bothered me

File: oss-banner.svg [hunk 2]
```diff
@@ -0,0 +1 @@
...wIDEgLjE0NC0uMTQ0aDYuNjgyMmEuMTQ0LjE0NCAwIDAgMSAuMTQ0LjE0NHptMC0yLjUzNjhhLjE0NC4xNDQgMCAwIDEtLjE0NC4xNDRIMy4yNTcxYS4xNDQuMTQ0IDAgMCAxLS4xNDQtLjE0NHYtLjk1MjNhLjE0NC4xNDQgMCAwIDEgLjE0NC0uMTQ0aDYuNjgyMmEuMTQ0LjE0NCAwIDAgMSAuMTQ0LjE0NHptMC0yLjUzNjhhLjE0NC4xNDQgMCAwIDEtLjE0NC4xNDRIMy4yNTcxYS4xNDQuMTQ0IDAgMCAxLS4xNDQtLjE0NHYtLjk1MjRhLjE0NC4xNDQgMCAwIDEgLjE0NC0uMTQ0aDYuNjgyMmEuMTQ0LjE0NCAwIDAgMSAuMTQ0LjE0NHptMTAuODAzNyA1LjA2OTZhLjE0NC4xNDQgMCAwIDEtLjE0NC4xNDRoLTYuNjgyM2EuMTQ0LjE0NCAwIDAgMS0uMTQ0LS4xNDR2LS45NTIzYS4xNDQuMTQ0IDAgMCAxIC4xNDQtLjE0NGg2LjY4MjJhLjE0NC4xNDQgMCAwIDEgLjE0NC4xNDR6bTAtMi41MzY4YS4xNDQuMTQ0IDAgMCAxLS4xNDQuMTQ0aC02LjY4MjNhLjE0NC4xNDQgMCAwIDEtLjE0NC0uMTQ0di0uOTUyM2EuMTQ0LjE0NCAwIDAgMSAuMTQ0LS4xNDRoNi42ODIyYS4xNDQuMTQ0IDAgMCAxIC4xNDQuMTQ0em0wLTIuNTM2OGEuMTQ0LjE0NCAwIDAgMS0uMTQ0LjE0NGgtNi42ODIzYS4xNDQuMTQ0IDAgMCAxLS4xNDQtLjE0NHYtLjk0ODRhLjE0NC4xNDQgMCAwIDEgLjE0NC0uMTQ0aDYuNjgyMmEuMTQ0LjE0NCAwIDAgMSAuMTQ0LjE0NHYuOTUyNHoiLz48L3N2Zz4="/><text transform="scale(.1)" x="621.25" y="175" textLength="662.5" fill="#fff">README 🦉</text><text transform="scale(.1)" x="2631.25" y="175" textLength="2877.5" fill="#fff" font-weight="bold">DEVELOPER HUBS THAT DEVELOPERS LOVE</text></g></svg>
```
Comment (erunion) [inline oss-banner.svg:1]: Rad that we can change this copy without help from design because it's an SVG

**Review by erunion — APPROVED** [review]

**Pull request merged**